bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBFHToWWSemilep_off_shell_4f_LO/VBFHToWWSemilep_off_shell_S_4f_LO/VBFHToWWSemilep_off_shell_S_4f_LO_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model loop_sm
+define l- = e- mu- ta-
+define l+ = e+ mu+ ta+
+generate p p > h > j j w+ w- $h, w+ > l+ vl, w- > l- vl~
```

Comment:
I just noticed that here you are defining fully leptonic decays instead of semi-leptonic.